src/runtime/GlobalObjectBuiltinTypedArray.cpp

```diff
@@ -1607,6 +1607,26 @@ static Value builtinTypedArrayToStringTagGetter(ExecutionState& state, Value thi
     return Value();
 }
 
+static Value builtinTypedArrayAt(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    // Let O be ? ToObject(this value).
+    if (!thisValue.isObject()) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, state.context()->staticStrings().ArrayIterator.string(), true, state.context()->staticStrings().next.string(), ErrorObject::Messages::GlobalObject_CalledOnIncompatibleReceiver);
+    }
+    Object* obj = thisValue.asObject();
```

Comment:
It seems that this part can be removed because `validateTypedArray` checks the type of thisValue in itself.